AIPscan/Aggregator/tasks.py

```diff
@@ -283,6 +291,43 @@ def package_lists_request(self, storage_service_id, timestamp, packages_director
     }
 
 
+def start_index_task(fetch_job_id):
+    task = index_task.delay(fetch_job_id)
+
+    index_task_obj = index_tasks(
+        index_task_id=task.id, fetch_job_id=fetch_job_id, indexing_start=datetime.now()
+    )
```

Comment:
The `workflow_coordinator_id` is the Celery-provided ID for the execution of the workflow coordinator task. The workflow coordinator task starts by spawning a package lists request task (requesting lists of packages from the storage service). The `package_task` table in the database used by Celery records both the `workflow_coordinator_id` and the ID of the package lists request task. The workflow logic is indeed a bit of a Rube Goldberg machine and it would be good, in a future PR, to simplify things if possible.